### PR TITLE
FIX - Adding Group to GenericWork

### PR DESCRIPTION
**Pull request by rbalekai**

### CODE REVIEW

**Comment by jcoyne** [comment]
Needs a test.

File: app/views/curation_concern/base/_linked_groups.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       </label>
     </span>
     <div class="controls">
-      <% prefix = f.object.class.to_s.downcase %>
+      <% prefix = f.object.class.to_s.underscore %>
```
Comment (jcoyne) [inline app/views/curation_concern/base/_linked_groups.html.erb]: Should be `ActiveModel::Naming.route_key(f.object.class)`

**Comment by rbalekai** [comment]
ActiveModel::Naming.route_key(f.object.class) didn't work. It converted GenericWork to 'generic_works'. But whats required is 'generic_work'

**Comment by jcoyne** [comment]
Use `ActiveModel::Naming.singular_route_key(Blog::Post)` then.

**Comment by rbalekai** [comment]
I will open the PR again after implementing Justin's suggestion

**Pull request closed without merging**